pkg/master/master.go

```diff
@@ -487,15 +487,5 @@ func DefaultAPIResourceConfigSource() *serverstorage.ResourceConfig {
 		admissionregistrationv1beta1.SchemeGroupVersion,
 	)
 
-	// all extensions resources except these are disabled by default
```

Comment:
clarify this for me.  I'd expect a server to come up with these enabled in a normal flow now.  True?

Reply:
if not set here, default they all enabled.

Reply:
> if not set here, default they all enabled.

ok, weird that the comment was backwards.

Reply:
I looked into this, and also test. remove it does not matter.